ChangeLog

```diff
@@ -279,6 +279,8 @@ Release date: tba
 
       Fixes #1031
 
+    * Let the user modify msg-template when Pylint is called from a Python script
+      Fixes #1269
```

Comment:
It needs one extra empty line between actual message and comment. See other entries in ChangeLog file for reference.